sources/packages/web/src/views/aest/SupportingUser.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <header-navigator
+    title="Back to student applications"
+    :routeLocation="{
+      name: AESTRoutesConst.STUDENT_APPLICATIONS,
+      params: { studentId },
+    }"
+    subTitle="Financial Aid Application"
+  >
+  </header-navigator>
+  <full-page-container class="my-2">
+    <formio
+      v-if="formName"
+      :formName="formName"
+      :data="formData"
+      :readOnly="true"
+    ></formio>
+    <Message severity="warn" :closable="false" v-else>
```

Comment:
v-else can be moved right after Message for readability. Initially I got confused until noted the v-else at the end.